.github/workflows/int-and-connected-test-run.yml

```diff
@@ -34,7 +34,7 @@ on:
       - '**.gradle'
       - 'Dockerfile'
       - 'datarepo-clienttests/**'
-      - '.github/workflows/int-and-connected-test-run.yml'
+      - '.github/workflows/**'
```

Comment:
I am not convinced of the value or need to run int / connected tests for unrelated workflow changes, outside of making a required action for merge go green.  In those cases, I would be comfortable with merging the PR as an admin, but am curious what your thoughts are.

Reply:
I'm not convinced either - my goal was to see if I was understanding this correctly and I think I do. I am generally opposed to special PRs that are allowed to skip through required CI checks - it just adds to the degree of tribal knowledge necessary to maintain a repo. There should be a way to check that if an action in't run due to a `paths` exclude, then it should not be marked as required. That's essentially my next test.